Cargo.lock

```diff
@@ -2,21 +2,6 @@
 # It is not intended for manual editing.
 version = 3
 
-[[package]]
```

Comment:
Did this previously depend on 2 different versions of actix-codec?  This file is autogenerated I assume?

Reply:
Yeah, Cargo.lock is autogenerated.  By my understanding, multiple versions of a library can exist in the project if they're being used in different places?  In this case, two different libraries were using two different versions of actix-codec and one of those versions was bad.

Reply:
That's neat that it can use both at once.  Java gets into trouble when that happens.